integration-tests/test/shared/utils.ts

```diff
@@ -95,8 +95,9 @@ export const fundUser = async (
 ) => {
   const value = BigNumber.from(amount)
   const tx = recipient
-    ? gateway.depositTo(recipient, { value })
-    : gateway.deposit({ value })
+    ? gateway.depositTo(recipient, 150_000, '0xFFFF', { value, gasLimit: 150_000 })
+    : gateway.deposit(150_000, '0xFFFF', { value, gasLimit: 150_000 })
+
   await waitForXDomainTransaction(watcher, tx, Direction.L1ToL2)
```

Comment:
The integration tests are not getting past this point. 
The error message is: 
```
 Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":146,\"error\":{\"code\":-32603,\"message\":\"Transaction reverted without a reason\"}}", error={"code":-32603}, requestBody="{\"method\":\"eth_sendRawTransaction\",\"params\":[\"0xf8f4318501dcd65000830249f094998abeb3e57409262ae5b751f60747921b33613e888ac7230489e80000b884e2cb45a600000000000000000000000000000000000000000000000000000000000249f000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000002ffff00000000000000000000000000000000000000000000000000000000000082f4f6a07f352d12d1f75ff89eb374bbe3b9f9b60a8d4347d14dbe1a33a4fea974c745c0a068706cea3887ee49ad40e699394f6538cbbb5258df77ab0af5dfeff7be2847a7\"],\"id\":146,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="http://l1_chain:8545", code=SERVER_ERROR, version=web/5.1.0)
      at Logger.makeError (/opt/optimism/node_modules/@ethersproject/logger/src.ts/index.ts:205:28)
      at Logger.throwError (/opt/optimism/node_modules/@ethersproject/logger/src.ts/index.ts:217:20)
      at /opt/optimism/node_modules/@ethersproject/web/src.ts/index.ts:284:28
      at step (/opt/optimism/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/opt/optimism/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/opt/optimism/node_modules/@ethersproject/web/lib/index.js:5:58)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

Look into the call stack, the error is happening here:

https://github.com/ethereum-optimism/optimism/blob/baf69efa4c5898f84863d0e5423f96af74afea89/integration-tests/test/shared/watcher-utils.ts#L45-L56 

ie. `await tx` does not resolve, and we get that error. 

Looking at the tx data in the error, I'm confused, as I cannot find any functions in our system with the signature 
`f8f43185`.

I would have expected this to be a call to the L2 messenger's `relayMessage` function, but it has the following sig: 
`cbd4ece9: relayMessage(address,address,bytes,uint256)`